src/templates/blogTemplate.js

```diff
@@ -0,0 +1,32 @@
+import React from "react"
+import { graphql } from "gatsby"
+
+export default function Template({ data }) {
+  const { markdownRemark } = data
+  const { frontmatter, html } = markdownRemark
+  return (
+    <div className="blog-post-container">
+      <div className="blog-post">
+        <h1>{frontmatter.title}</h1>
+        <h2>{frontmatter.date}</h2>
+        <div
+          className="blog-post-content"
+          dangerouslySetInnerHTML={{ __html: html }}
+        />
+      </div>
+    </div>
```

Comment:
Need to make this Layout more prettified. Like with a site header and such. Waaaay to wide, also!

Reply:
Made it nice! ✨